server/auth.js

```diff
@@ -21,6 +21,18 @@ passport.deserializeUser(
   }
 );
 
+// Local signup
+auth.post('/local/signup', (req, res, next) => {
+  User.create(req.body)
+  .then(user => {
+    req.login(user, (err) => {
```

Comment:
Does this attach the user to the req.session object? (Asking because I  totally forget.) If not, can we do that manually so that the front-end can do its thing with that info?